docs/api/annotations.md

```diff
@@ -0,0 +1,233 @@
+# Annotations
+
+An annotation assigns pronunciation errors to specific fragments of an audio recording performed the
+an annotator user.
+
+## List all annotations
+
+A user with the 'ADMIN' role will receive all the annotations.
+Non-admins will receive only the annotations assigned to them.
+
+### URL
+
+```http
+GET /annotations HTTP/1.1
+```
+
+### Request parameters
```

Comment:
Not sure but my guess is that the parameters you mention are actually more commonly called `filters` in the docs? Also very unclear on how to use them (that made me actually assume they are filters).